docs/standard-library/built-in-data-types.md

```diff
@@ -99,13 +99,13 @@ model OptionalProperties<T>
 
 
 ```typespec
-model Record<T>
+model Record<Property>
 ```
 
 #### Template Parameters
 | Name | Description |
 |------|-------------|
-| T | The type of the properties |
+| Property | The type of the properties |
```

Comment:
Rec: `PropertyType`. I realize we don't use type anywhere else, but this make it seem like it's a property reference.